src/async-status-indicator/index.js

```diff
@@ -1,2 +1,4 @@
-import AsyncStatusIndicator from './AsyncStatusIndicator';
-export default AsyncStatusIndicator;
\ No newline at end of file
+import AsyncStatusIndicator, { Props as AsyncStatusIndicatorProps, AsyncStatusIndicatorState } from './AsyncStatusIndicator';
+
+export { AsyncStatusIndicatorProps, AsyncStatusIndicatorState };
+export default AsyncStatusIndicator;
```

Comment:
I confirm that the one liner is working, but in the end there's no a strict guideline about how to do that. (There are mixed usages in the app)